app/signals/apps/api/urls.py

```diff
@@ -103,6 +104,7 @@
         re_path(r'signals/(?P<uuid>[-\w]+)/attachments/?$',
                 PublicSignalAttachmentsViewSet.as_view({'post': 'create'}), name='public-signals-attachments'),
         re_path(r'questions/?$', PublicQuestionViewSet.as_view({'get': 'list'}), name='question-detail'),
+        path('verify-email', EmailVerificationView.as_view(), name='email-verification'),
```

Comment:
Naming the endpoint `reporter/verify-email` instead of `verify-email` will provide a clearer indication that it is used to verifying the email of a reporter (melder).